src/util/rocksdb.c

```
@@ -0,0 +1,111 @@
+	rocksdb_flush(db, flush_options, &err);
+	ASSERT(!err);
+}
+
+char *RocksDB_get(const char *key) {
+	char *err = NULL;
+	size_t len;
+	char *returned_value = rocksdb_get(db, readoptions, key, strlen(key), &len, &err);
+	ASSERT(!err);
+	char *res = rm_strdup(returned_value);
+	free(returned_value);
+	return returned_value;
```

Cppcheck (2.10-2)

[error] Returning/dereferencing 'returned_value' after it is deallocated / released

(deallocret)